specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/bigDataPool.json

```
@@ -579,6 +580,7 @@
           "description": "The maximum number of executors alloted",
           "x-ms-mutability": [
             "create",
+            "read",
             "update"
           ]
```

Rather than adding "read" you should just remove x-ms-mutability altogether, since [it's equivalent](https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-mutability:~:text=applying%20the%20extension%20as%20%22x%2Dms%2Dmutability%22%3A%20%5B%22create%22%2C%20%22read%22%2C%20%22update%22%5D%20or%20not%20applying%20it%20will%20have%20the%20same%20effect.) and less verbose
```suggestion
          "description": "The maximum number of executors alloted"
```